test/base.py

```diff
@@ -45,33 +48,36 @@ def flatten(l, ltypes=(list, tuple)):
         i += 1
     return ltype(l)
 
+
 class ColumnName(str):
     pass
 
+
 def dbparam(x):
     if x is None:
         return 'NULL'
     elif isinstance(x, ColumnName):
         return "`" + x + "`"
-    elif isinstance(x, basestring):
+    elif isinstance(x, bytes):
```

Comment:
bytes and str are different in python 3 so that won't work if the expectation is that this is looking for a string. If you are using future you need to do
```python
from past.builtins import basestring
```